content/metadata.yaml

```diff
@@ -1,21 +1,29 @@
 ---
-title: "Manubot Rootstock: Manuscript Title"
+title: "The probability of edge existence due to node degree: a baseline for link prediction"
```

Comment:
Would "a baseline for link prediction" be better as "a baseline for network-based predictions"? Probably helpful to orient readers to networks?

Reply:
I prefer your suggestion